follow.go

```diff
@@ -215,7 +215,8 @@ func (f *Follow) list(roles scope) ([]Follow, Jerror) {
     left join score s on s.siret = f.siret and s.libelle_liste = $2
 		left join entreprise_ellisphere0 g on g.siren = en.siren
 		left join terrind ti on ti.code_commune = et.code_commune
-    where f.username = $3 and f.active
+		where f.username = $3 and f.active
+		order by f.id
```

Comment:
J'ai ajouté un ordre à l'affichage des suivis, il n'y en avait pas et c'est ce qui explique pourquoi le test sautait après que j'ai mis en place la vue pour les codes naf, comme de nouveaux indexes sont rentrés dans la danse le tri de la liste se faisait désormais également grâce à ces indexes et est un peu différent de celui d'origine. Le contenu de la liste des suivis n'avait pas été altéré hormis l'ordre.

Ici j'ai fait en sorte que les suivis soient affichés avec les plus anciens au début, si tu préfères que ce soit dans l'autre sens il suffit de faire ça:
```suggestion
		order by f.id desc
```

Je trouve que mettre les plus anciens en premier peut inciter les utilisateurs à faire le ménage, c'est un peu un dark pattern, mais c'est pour le bien ! ;)